utils/prepare-artifacts.sh

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+print_help() {
+    echo "Usage: utils/prepare-artifacts.sh OPTIONS"
+    echo ""
+    echo "Options:"
+    echo " --[no-]reset                 Clear the pkg directory or not (default: no)"
+    echo " --[no-]delete-zips           Delete the downloaded zip files or not (default: no)"
+    echo " --[no-]download-artifacts    Download the artifacts or not (default: no)"
+    echo " --api-token TOKEN            The Github API token for downloading artifacts"
+    echo " --run-id ID                  The id of the Workflow run for which to download artifacts"
+}
+
+GITHUB_API_TOKEN=
+RUN_ID=
+TARGET_DIR="target/pkg"
+DO_RESET=false
+DELETE_ZIPS=false
+DOWNLOAD_ARTIFACTS=false
+ZIP_DOWNLOADS_FOLDER="$TARGET_DIR/zips"
+
+
+while [[ $# -gt 0 ]]; do
+    case $1 in
+        --reset)
+            DO_RESET=true
+        ;;
+        --no-reset)
+            DO_RESET=false
+        ;;
+        --delete-zips)
+            DELETE_ZIPS=true
+        ;;
+        --no-delete-zips)
+            DELETE_ZIPS=false
+        ;;
+        --download-artifacts)
+            DOWNLOAD_ARTIFACTS=true
+        ;;
+        --no-download-artifacts)
+            DOWNLOAD_ARTIFACTS=false
+        ;;
+        --api-token)
+            GITHUB_API_TOKEN="$2"
+            shift
+        ;;
+        --run-id)
+            RUN_ID="$2"
+            shift
+        ;;
+        --help)
+            print_help
+            exit 0
+        ;;
+    esac
+    shift
+done
+
+if [ "$DOWNLOAD_ARTIFACTS" = true ]; then
+    if [ -z "$GITHUB_API_TOKEN" ]; then
+        echo "ERROR: Missing Github API token"
+    fi
+
+    if [ -z "$RUN_ID" ]; then
+        echo "ERROR: Missing github workflow run id"
+    fi
+    print_help
```

Comment:
I think this will always print the help and exit if `DOWNLOAD_ARTIFACTS` is `true`